pom.xml

```diff
@@ -357,5 +357,9 @@
             <name>Ayelet Levron</name>
             <email>Ayelet.Levron@jpmchase.com</email>
         </developer>
+        <developer>
```

Comment:
don't you need to bump up the version as well?

Reply:
No - it is done automatically after merge to master